codecov.yml

```diff
@@ -0,0 +1,15 @@
+codecov:
+  require_ci_to_pass: yes
+
+coverage:
+  range: 80..100
+  round: down
+  precision: 2
+
+  status:
+    project:                   # measuring the overall project coverage
+      default:                 
+        enabled: no           
```

Comment:
I think enabling it already might be a good choice. Maybe with some leniency until people gets used to it like in example "Ease Target Coverage" https://docs.codecov.com/docs/common-recipe-list